src/cli.c

```diff
@@ -453,7 +468,10 @@ main(int argc, char** argv)
 
    if (do_free)
    {
-      free(password);
+      if (password)
```

Comment:
You don't need to guard a `free()` - it understands a `NULL` value

Reply:
Got it.